src/lib/journey/journey.store.ts

```diff
@@ -4,72 +4,57 @@ import {
   FRLoginFailure,
   FRLoginSuccess,
   StepType,
-  TokenManager,
-  UserManager,
 } from '@forgerock/javascript-sdk';
-import { get, writable, type Writable } from 'svelte/store';
+import type { Step, StepOptions } from '@forgerock/javascript-sdk/lib/auth/interfaces';
+import { writable, type Writable } from 'svelte/store';
 
-import type { StringDict } from '$lib/interfaces';
 import { htmlDecode } from '$lib/journey/utilities/decode.utilities';
-import { email, isAuthenticated, fullName } from '$lib/user/user.store';
 
-export interface InitObject {
-  step: Writable<StepTypes>;
-  getStep: (prevStep?: StepTypes) => Promise<void>;
-  failureMessage: Writable<string | null>;
-  submittingForm: Writable<boolean>;
+export interface JourneyStore extends Pick<Writable<JourneyStoreValue>, 'subscribe'> {
```

Comment:
I converted the export from a regular function to a custom store (observable). This helps leverage Svelte's own features and normalizes the stores' interfaces.